socialhome/streams/app/components/AuthorBar.vue

```diff
@@ -94,17 +93,17 @@ export default Vue.component("author-bar", {
                 console.error("Not logged in")
                 return
             }
-            this.$http.post(`/api/profiles/${this.currentBrowsingProfileId}/remove_follower/`, {guid: this.guid})
-                .then(() => { this.$data._isUserFollowingAuthor = false })
+            this.$http.post(`/api/profiles/${this.currentBrowsingProfileId}/remove_follower/`, {guid: this.author.guid})
+                .then(() => { this.author.isUserFollowingAuthor = false })
```

Comment:
This works and updates the store value even though `this.author` is a `computed` value?

Reply:
Yep :)